auth/auth.go

```diff
@@ -560,10 +568,12 @@ func (tp tokenProvider2LO) Token(ctx context.Context) (*Token, error) {
 		return nil, err
 	}
 	req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
+	tp.logger.DebugContext(ctx, "2LO token fetch", "request", internallog.HTTPRequest(req, []byte(v.Encode())))
```

Comment:
Bikeshed: "request" instead of "fetch" in all of these request logs. (Logic: request/response are the standard pair, fetch/response seems unusual.)

Reply:
Done.